src/backend/gl/src/window/egl.rs

```diff
@@ -142,7 +142,12 @@ impl hal::Instance<crate::Backend> for Instance {
             Ok(Some(config)) => config,
             Ok(None) => {
                 log::warn!("no compatible EGL config found, trying off-screen");
-                supports_native_window = false;
+                // Android surface still presentable
+                supports_native_window = if cfg!(target_os = "android") {
```

Comment:
This seems rather concerning. Which of the `config_attributes` does Android not like, to start with?

Reply:
I'll try to figure it out later